app/src/main/AndroidManifest.xml

```diff
@@ -8,7 +8,7 @@
          location permissions for the "MyLocation" functionality.
     -->
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
-
+    <uses-permission android:name="android.permission.INTERNET" />
```

Comment:
Added Internet permission per the suggestion here: https://developer.android.com/reference/android/Manifest.permission.html#INTERNET

Reply:
Is this required for anything that the app is currently doing?  More than likely, as most apps do require networking permission.  As a general rule, don't add permissions that are not needed.  Your users will wonder why you need that permission.